source/agora/common/EnrollmentManager.d

```diff
@@ -79,7 +80,7 @@ public class EnrollmentManager
         this.db.execute("CREATE TABLE IF NOT EXISTS node_enroll_data " ~
             "(key CHAR(128) PRIMARY KEY, val BLOB NOT NULL)");
 
-        // set node's KeyPair object
+        // create Pair object from KeyPair object
         this.key_pair.v = Scalar(key_pair.secret);
```

Comment:
I think we need to replace this with the newly merged function, don't we ?

Reply:
I will do that on another PR. Can you agree with that?